src/contracts/core/DelegationManager.sol

```diff
@@ -939,6 +993,28 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         return approverDigestHash;
     }
 
+    /**
+     * @notice Calculates the digest hash to be signed by an operator to register with an AVS
+     * @param operator The account registering as an operator
+     * @param avs The AVS the operator is registering with
+     * @param salt A unique and single use value associated with the approver signature.
+     * @param expiry Time after which the approver's signature becomes invalid
+     */
+    function calculateOperatorRegistrationDigestHash(
+        address operator,
+        address avs,
+        bytes32 salt,
+        uint256 expiry
+    ) public view returns (bytes32) {
```

Comment:
Request to shorten this function name. I see it's similar to other methods in this contract... not sure whether we need a second pass to just shorten all the names (similar to the middleware renaming PRs), or what.